src/Microsoft.ML.Experimental/Experimental.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.ML.Experimental
+{
+    public static class Experimental
+    {
+        /// <summary>
+        /// Stop the exeuction of pipeline in <see cref="MLContext"/>
+        /// </summary>
+        /// <param name="ctx"></param>
+        public static void StopExecution(this MLContext ctx) => ctx.StopExecution();
```

Comment:
Do we also need to know if the `MLContext` has been canceled?

Reply:
Right now this is a synchronous operation that just turns on a flag for cancellation. Later when we implement mechanisms that actually make use of this flag to cancel a pipeline then we can check if a pipeline has been canceled may be by checking for an exception or something,

---
In reply to: [264861512](https://github.com/dotnet/machinelearning/pull/2797#discussion_r264861512) [](ancestors = 264861512)